composer.json

```diff
@@ -111,6 +111,7 @@
         "sonata-project/seo-bundle": "^2.5",
         "symfony/browser-kit": "^4.4 || ^5.2",
         "symfony/http-client": "^4.4 || ^5.2",
+        "symfony/messenger": "^4.4 || ^5.3",
```

Comment:
time to bump 5.2 for others too ?

Reply:
I can do it on next Pr, but on 3.x it does not seem super necessary since it is not full sf 5 support, only a few components. On 4.x is already on 5.3 for all.

Reply:
Maybe it's better to add support for symfony messenger 5.2 then ? for consistency

Reply:
If it is something important I can create another Pr for 3.x to up the versions of the other components.